### PR TITLE
Add `-keep_latest_n_logs` flag to purge old log files.

### PR DESCRIPTION
**Pull request by adsr**
Handy in combination with `-log_rotate_max_size` if you
want to keep around N log files of a certain size. This is
an alternative to `-keep_logs` which purges by timestamp.
All three flags can be active at the same time.

(It would be nice if glog exposed `MaxDuration` in addition to `MaxSize`.)

### CODE REVIEW

File: go/vt/logutil/purge.go [hunk 1]
```diff
@@ -74,22 +68,87 @@ func purgeLogsOnce(now time.Time, dir, program string, keep time.Duration) {
...
+// PurgeLogs removes old log files according to `-keep_logs` and
+// `-keep_latest_n_logs` flags. Current log files are never removed.
 func PurgeLogs() {
 	f := flag.Lookup("log_dir")
 	if f == nil {
 		panic("the logging module doesn't specify a log_dir flag")
 	}
 
-	if *keepLogs == 0*time.Second {
+	if *keepLogs == 0*time.Second && *keepLatestNLogs == 0 {
```
Comment (sougou) [inline go/vt/logutil/purge.go]: No need for `time.Second`. Just `0` should do. Same comment below. It's a go convention related to zero values.

**Comment by sougou** [comment]
@adsr ping (conflicts need merging).
Also, the DCO failed. So, it may be better to create a new PR because it doesn't go green even if you fix the commit.

**Comment by adsr** [comment]
I'm ok to close this PR. We are using `-keep_logs_by_mtime` instead.

**Comment by sougou** [comment]
Did the mtime work for you? I recently found a bug where the purge would happen only once: #5012

**Pull request closed without merging**